cleancode/src/main/java/com/epam/cleancode/singleresponsibility/Calculator.java

```diff
@@ -1,48 +1,52 @@
 package com.epam.cleancode.singleresponsibility;
 
-public class Calculator{
+public class Calculator {
 
     private static final String SEPARATOR = ",";
+    private static final String REGULAR_EXPRESSION = "[\\d" + SEPARATOR + "]*";
```

Comment:
please rename it. I see that it's regular expression, but what exactly this regular expression can find?